packages/cli/core/compiled/ajv-keywords/ajv/dist/types/jtd-schema.d.ts

```diff
@@ -34,7 +34,7 @@ export declare type SomeJTDSchemaType = (// ref
 declare type RequiredKeys<T> = {
     [K in keyof T]-?: undefined extends T[K] ? never : K;
 }[keyof T];
-/** optional or undifined-able keys of an object */
+/** optional or undefined-able keys of an object */
```

Comment:
`compiled` directory contains the code of pre-bundled dependencies, we should not manually modify the code in this directory.

Reply:
Yes, I reversed the changes to the three files in the `compiled` directory.